### DISCUSSION TITLE
fits_open_diskfile error

### DISCUSSION THREAD

**Discussion by CoffeebarrelVFX** [deepskystacker/DSS#208] [Q&A]
![Screenshot 2024-07-07 100639](https://github.com/deepskystacker/DSS/assets/174948314/3ffe4e92-65cf-45e7-84e9-e0c0ce5fd7f3)
getting this error on version 4.2.6

**Comment by perdrix52** [comment]
.arw is an extension for Sony raw files.   The code tried to open it as a raw file and didn’t recognise the file because the version of libraw that 4.2.6 uses doesn’t recognise that file as a valid raw file.   In that case, the code falls through to try to open it as a FITS file and fails.  What camera do you have? Have you tried the latest version (5.1.6)?

 

David

**Comment by perdrix52** [comment]
PS please follow the instructions to report problems on the mailing list at groups.io
